libs/filamentapp/include/filamentapp/FilamentApp.h

```diff
@@ -73,8 +73,8 @@ class FilamentApp {
 
     void setDropHandler(DropCallback handler) { mDropHandler = handler; }
 
-    void run(const Config& config, SetupCallback setup, CleanupCallback cleanup,
-            ImGuiCallback imgui = ImGuiCallback(), PreRenderCallback preRender = PreRenderCallback(),
+    void run(const Config& config, const SetupCallback& setup, const CleanupCallback& cleanup,
+            ImGuiCallback imgui = ImGuiCallback(), const PreRenderCallback& preRender = PreRenderCallback(),
```

Comment:
We appreciate this change, but it's not related to this PR -- so I would prefer to revert it here. It's just for clarity and not be confused later.

Reply:
removed const.